fighthealthinsurance/models.py

```
@@ -201,12 +201,14 @@ def get_form(self):
             else:
                 return None
         else:
+            import fighthealthinsurance.forms.questions
```

Ruff (0.8.2)

`fighthealthinsurance.forms.questions` imported but unused

Remove unused import: `fighthealthinsurance.forms.questions`

(F401)